flask/main.py

```diff
@@ -423,7 +430,12 @@ def webhook_post():
        Uses `find_location_and_meal` or `find_item` intent handlers and returns appropriate
        JSON response.
     """
+    secrets = get_secrets()
+    dba = dashbotgoogle.google(secrets.get('dashbot_api'))
```

Comment:
Can we make it so that if this value isn't found we don't do anything? 
Like

```
dba_key = secrets.get('dashbot_api')
dba = False
if dba_key:
        dba = dashbotgoogle.google(secrets.get('dashbot_api'))
```

Then later also checking before we send it in the code below. Then we can disable this code if we don't want it sending (like in a test environment)

Maybe just
`dba and dba.logincoming(req_data) `